### ISSUE TITLE
Typo in true airspeed calculation

### ISSUE THREAD

**Issue by lj-at-work** [mitre/FMACM#28]
Hey, it looks like there's a copy/paste error in the `AircraftState` TAS implementation:

```c++
Units::Speed AircraftState::GetTrueAirspeed() const {
   Units::MetersPerSecondSpeed tas_x, tas_y;
   tas_x = Units::FeetPerSecondSpeed(m_xd) - Units::MetersPerSecondSpeed(m_Vwx);
   tas_y = Units::FeetPerSecondSpeed(m_xd) - Units::MetersPerSecondSpeed(m_Vwy);
   Units::MetersPerSecondSpeed tas(hypot(tas_x.value(), tas_y.value()));
   return tas;
}
```

In particular

```c++
  tas_y = Units::FeetPerSecondSpeed(m_xd) - Units::MetersPerSecondSpeed(m_Vwy);
```
should be

```c++
  tas_y = Units::FeetPerSecondSpeed(m_yd) - Units::MetersPerSecondSpeed(m_Vwy);
```

(I'm not sure there isn't also a more general question of the z-component but I realize that it might be used contingently, when you expect level flight where the angle of attack is close to zero)

**Comment by sbowman-mitre** [comment]
Sorry for the slow response. Thanks for posting this bug! This has been fixed internally. You'll see the fix in the v3.5.3 release at the end of the summer.

**Comment by sbowman-mitre** [comment]
This has been fixed the v3.5.3 tag. Thanks!
